### ISSUE TITLE
Entering debug mode results in double "[DBG]: " tag

### ISSUE THREAD

**Issue by daleecc** [PowerShell/vscode-powershell#863]
### System Details

- Operating system name and version: Windows 7 (64-bit)

```
PS > code -v
1.13.0
376c52b955428d205459bea6619fc161fc8faacf
PS > $pseditor.EditorServicesVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
1      3      1      0


PS > code --list-extensions --show-versions
DotJoshJohnson.xml@1.8.0
ms-mssql.mssql@1.0.0
ms-vscode.csharp@1.10.0
ms-vscode.PowerShell@1.3.1
ms-vsts.team@1.118.2
robertohuertasm.vscode-icons@7.9.0
PS > $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.0.10586.117
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.10586.117
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

When a breakpoint is hit in debug mode, the command prompt includes two "[DBG]: " tags at the beginning. For example:

```
[DBG]: [DBG]: PS C:\>
```

**Comment by daviwil** [comment]
Yep, somehow that slipped in there.  Thanks for filing!

**Comment by tylerl0706** [comment]
Fixed in https://github.com/PowerShell/PowerShellEditorServices/pull/627.

Will be released in 1.6.0
